Core/Utilities/DErrno.py

```diff
@@ -10,35 +10,79 @@
 
     It also contains utilities to manipulate these errors.
 
-    Finaly, it contains a DErrno class that contains an error number
+    Finally, it contains a DErrno class that contains an error number
     as well as a low level error message. It behaves like a string for
     compatibility reasons
+
+    In order to add extension specific error, you need to create in your extension the file
+    Core/Utilities/DErrno.py, which will contains the following dictionnary:
```

Comment:
will contains -> will contain
and
"dictionary" (c'est pas "dictionnaire" :) )

Anyway this should be added to the DiracDOCS. And don't forget to update https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-v6r15